tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
+
 	private $tmpDir;
 
+	protected function getPort()
+	{
+		return $this->startPort + $this->triedPorts;
+	}
+
+	protected function getIPAddress()
+	{
+		$records = dns_get_record(php_uname('n'), DNS_A);
+
+		if(!empty($records))
```

Comment:
My CS wants spaces after language constructs and before (, like: ```if (```). There are many occurrences like this, so please fix there.